phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
I don't understand the comment "add the enctype attribute to the template".  This is not meant to be there always, if it were, we would be better off hardcoding it (as in https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/adm/style/acp_users_avatar.html#L1).  This is meant to be filled in by extensions needing it, or (as in my case) by new profile field types. But not hardcoding/always present.
And no, I am not going to change so many places all around phpbb.  I'd rather keep these as they are now.